### PR TITLE
Review-past: milestone1

### PR DESCRIPTION
**Pull request by ambrose-kibet**
### In this milestone we: 
- create index.hml file
- add styling with style.css

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,121 +1,5 @@
...
-- [⭐️ Show your support](#support)
-- [🙏 Acknowledgements](#acknowledgements)
-- [📝 License](#license)
-
-<!-- PROJECT DESCRIPTION -->
-
-# 📖 Microverse Trials <a name="about-project"></a>
-
-**Microverse Trials** Is a microverse trials free code camp project.
-
```
Comment (ambrose-kibet) [inline README.md:33]: - [ ] please add a more decriptive readme about your project.This  helps other users understand what your project is about.

File: js/index.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    from red to green.
+  - As you start to build out your 
+    project, when tests are failing, 
+    you should get helpful errors 
+    along the way!
+    ************/
+
+// PLEASE NOTE: Adding global style rules using the * selector, or by adding rules to body {..} or html {..}, or to all elements within body or html, i.e. h1 {..}, has the potential to pollute the test suite's CSS. Try adding: * { color: red }, for a quick example!
+
+// Once you have read the above messages, you can delete all comments.
```
Comment (ambrose-kibet) [inline js/index.js:27]: - [ ]  please remove comment on your code  as they make your codebase messy

**Review by ambrose-kibet — COMMENTED** [review]
## What you did right:
- Added a discriptive readme.file.
- used Github flow on your project

## Changes required 
Please make the requested changes below on the comments.
Submit for a re-review when youre done.

**Review by amejid — APPROVED** [review]
Hi @ambrose-kibet, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- Correct use of Github features :heavy_check_mark: 

## Optional suggestions
- Kindly use emojis and gifs in your reviews to make them human-friendly. 😎️🤝️

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**